www/json/trip_confirm_options.json

```diff
@@ -0,0 +1,30 @@
+{
+  "MODE": [
```

Comment:
are you using the trip confirm functionality? If not, I would omit this - it just causes additional confusion and potential future merge conflicts. There is already a `trip_confirm_options.json.sample` to provide an overview.

Reply:
I just put the file to avoid 404 error when debugging the phone.

Reply:
you can ignore the 404 error; there is a fallback to read the sample file in that case and everything still works

Reply:
Ok but how can I un-commit it?

Reply:
`git rm --cached www/json/trip_confirm_options.json`
and then `git commit`

Reply:
I thought there is a magic command that you can scoop the file out of commit history. Anyways, done.